x/iro/types/params.go

```diff
@@ -18,10 +19,16 @@ var (
 	DefaultMinLiquidityPart                             = "0.4"                       // default: at least 40% goes to the liquidity pool
 	DefaultMinVestingDuration                           = 7 * 24 * time.Hour          // default: min 7 days
 	DefaultMinVestingStartTimeAfterSettlement           = 0 * time.Minute             // default: no enforced minimum by default
+
+	DefaultFairLaunch = FairLaunch{
+		AllocationAmount: math.NewInt(1e9).MulRaw(1e18),                      // 1B RA tokens
+		TargetRaise:      sdk.NewCoin("adym", math.NewInt(2e4).MulRaw(1e18)), // 20K DYM
```

Comment:
there is a const for adym in `github.com/dymensionxyz/dymension/v3/app/params`
```suggestion
		TargetRaise:      sdk.NewCoin(params.BaseDenom, math.NewInt(2e4).MulRaw(1e18)), // 20K DYM
```